Bottom: cedb74b5b2b3ebaea421918e167da22559aff4ef
Top:    2ce12d4d62f7b8a74f95d66900c4459585768cda
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-12-19 01:00:18 +0000

hid/gerber: Fix a couple of typos in drill_sort routine

Should fix sorting of drill holes by coordinate.

(Caught by clang)


---

diff --git a/src/hid/gerber/gerber.c b/src/hid/gerber/gerber.c
index ce20e3b..3930941 100644
--- a/src/hid/gerber/gerber.c
+++ b/src/hid/gerber/gerber.c
@@ -628,8 +628,8 @@ drill_sort (const void *va, const void *vb)
   if (a->diam != b->diam)
     return a->diam - b->diam;
   if (a->x != b->x)
-    return a->x - a->x;
-  return b->y - b->y;
+    return a->x - b->x;
+  return a->y - b->y;
 }
 
 static int
